sv_pipeline/genome/utils/download_utils.py

```diff
@@ -4,7 +4,12 @@
 import tempfile
 from contextlib import contextmanager
 import hail as hl
-from tqdm import tqdm
+
+try:
```

Comment:
This is gross.  Other options:
1) Just add `tqdm` to the requirements (at the time this seemed worse to me but 🤷 ).
2) Just remove it.

Reply:
I went with option 1 after re-considering.